cdap-cli/src/main/java/co/cask/cdap/cli/command/metrics/GetMetricCommand.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.cli.command.metrics;
+
+import co.cask.cdap.cli.CLIConfig;
+import co.cask.cdap.cli.util.AbstractAuthCommand;
+import co.cask.cdap.cli.util.ArgumentParser;
+import co.cask.cdap.cli.util.RowMaker;
+import co.cask.cdap.cli.util.table.Table;
+import co.cask.cdap.client.MetricsClient;
+import co.cask.cdap.proto.MetricQueryResult;
+import co.cask.common.cli.Arguments;
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.google.inject.Inject;
+
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Gets the value of a metric.
+ */
+public class GetMetricCommand extends AbstractAuthCommand {
+
+  private final MetricsClient client;
+
+  @Inject
+  public GetMetricCommand(MetricsClient client, CLIConfig cliConfig) {
+    super(cliConfig);
+    this.client = client;
+  }
+
+  @Override
+  public void perform(Arguments arguments, PrintStream output) throws Exception {
+    String metric = arguments.get("metric-name");
+    Map<String, String> tags = ArgumentParser.parseMap(arguments.get("tags", ""));
+    String start = arguments.get("start", "");
```

Comment:
Not right now. Need to change `Arguments` to support that, but it's in a separate repo.